src/CommunityToolkit.Maui/Converters/StateToBooleanConverter.shared.cs

```diff
@@ -1,13 +1,30 @@
 ﻿using System;
 using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
-using CommunityToolkit.Maui.UI.Views;
-using Microsoft.Maui.Controls;
 
 namespace CommunityToolkit.Maui.Converters;
 
+/// <summary>State used by <see cref="StateToBooleanConverter"/> </summary>
+public enum LayoutState
```

Comment:
That's fair - we can create a folder called `Enums`.

We have a lot of single-use enums (e.g. Only `StateToBooleanConverter` uses the `LayoutState`) that were sporadically placed (some were in the `Converters` folder, some were in the `Views` folder etc), which is why I moved the single-use enum into the class that uses it.

Reply:
Hey Vlad! I started the process of creating an `Enums` folder and moving all of the enums to it. 

But then I quickly realized that this would require a new `xmlns`, making life harder for XAML devs.

I agree that we should move the files, but I think we should wait until #144 is completed to avoid making life hard for XAML devs before #144 makes it simpler.

I've added this task to a Proposal #219, which we'll complete after #144 👍